src/components/home-layout/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { twMerge } from 'tailwind-merge';
+
+interface Props {
+  contentIsVisible: boolean;
+  sidebar: React.ReactNode;
+  content: React.ReactNode;
+}
+
+const HomeLayout: React.FC<Props> = ({
+  contentIsVisible,
+  sidebar,
+  content,
+}) => {
+  return (
+    <div className="w-full min-h-screen flex">
+      <div
+        className={twMerge(
+          'transition-all duration-500 ease-in-out overflow-hidden lg:basis-2/6',
```

Comment:
Instead of using `flex-basis`, I would use `grid`, because you always have to resize manually. At the same time, if you hide the sidebar, the page will not be full width